src/errorpage.cc

```diff
@@ -180,20 +180,30 @@ error_hard_text[] = {
         ERR_REQUEST_START_TIMEOUT,
         "request start timedout"
     },
+    {
+        ERR_REQUEST_PARSE_TIMEOUT,
+        "request parse timedout"
+    },
+    {
+        ERR_RELAY_REMOTE,
+        "relay server response"
+    }
+}};
+
+/// error messages that may be configured/customized by the admin,
+/// like the regular ones, but that also have a hard-coded last-resort default
+static std::array<HardCodedMessage, 1> TemplateDefaults = {{
```

Comment:
Please rename this array.

```suggestion
static std::array<HardCodedMessage, 1> ErrorsWithDefaultTemplates = {{
```

Reply:
Done.